### PR TITLE
confirm box

### PR DESCRIPTION
**Pull request by hard75**
formato amount

spacing into the confirm box

### CODE REVIEW

File: src/components/liquality/ConfirmLiqualityTransaction.vue [hunk 1]
```diff
@@ -285,6 +293,7 @@ export default class ConfirmLiqualityTransaction extends Vue {
 
   async created() {
     this.rawTx = await this.txBuilder.getUnsignedRawTx(this.pegInTxState.normalizedTx);
+    this.bitcoinPrice = this.pegInTxState.bitcoinPrice;
```
Comment (josedahlquist) [inline src/components/liquality/ConfirmLiqualityTransaction.vue:296]: Are you using this bitcoinPrice field?

File: src/components/liquality/ConfirmLiqualityTransaction.vue [hunk 2]
```diff
@@ -177,11 +173,18 @@ import LiqualityTxBuilder from '@/middleware/TxBuilder/LiqualityTxBuilder';
...
   txId = '';
 
   rawTx = '';
 
+  bitcoinPrice = 0;
+
+  fixedDecimals = 2;
+
+  allOutputs = [];
+
```
Comment (josedahlquist) [inline src/components/liquality/ConfirmLiqualityTransaction.vue:187]: I don't see these fields been used. Do you need them?

**Review by josedahlquist — DISMISSED** [review]

File: src/components/liquality/ConfirmLiqualityTransaction.vue (hunk 1 above)
Comment (hard75) [reply src/components/liquality/ConfirmLiqualityTransaction.vue]: no, i delete it

File: src/components/liquality/ConfirmLiqualityTransaction.vue (hunk 2 above)
Comment (hard75) [reply src/components/liquality/ConfirmLiqualityTransaction.vue]: no, deleted it

**Review by josedahlquist — APPROVED** [review]

**Review by alexjavabraz — APPROVED** [review]
LGTM

**Review by leoiovlabs — APPROVED** [review]
LGTM

**Pull request merged**